src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.helpers.ts

```diff
@@ -114,7 +114,7 @@ export const getDetailsPanelData = function (
     }
   });
 
-  max = Math.ceil(max);
```

Comment:
Limits are still calculated using the max value for all tissues for a given gene and are still centered around 0, but scaling the max value slightly rather than rounding to the nearest integer ensures that tissues with small values are more clearly displayed